src/cupbearer/data/transforms.py

```diff
@@ -40,22 +42,36 @@ def __call__(self, sample):
 
 
 class ToTensor(AdaptedTransform):
-    def __img_call__(self, img):
+    def __img_call__(self, img: torch.Tensor) -> torch.Tensor:
```

Comment:
`img` is a PIL image here, right?

Reply:
Ah, yes `Union[PILImage, np.ndarray]` to be exact https://github.com/pytorch/vision/blob/main/torchvision/transforms/functional.py#L127